### PR TITLE
When computing Interpreter VMStack, push Value type instead of value …

### PR DESCRIPTION
**Pull request by ksh8281**
…stack allocated size

### CODE REVIEW

**Comment by ksh8281** [comment]
This may fixes https://github.com/Samsung/walrus/issues/97

File: src/parser/WASMParser.cpp [hunk 1]
```diff
@@ -74,6 +74,24 @@ struct WASMCodeInfo {
...
+        case F32:
+            return Walrus::Value::Type::F32;
+        case I64:
+            return Walrus::Value::Type::I64;
+        case F64:
+            return Walrus::Value::Type::F64;
+        case V128:
+            return Walrus::Value::Type::V128;
+        default:
+            return Walrus::Value::Type::Void;
```
Comment (clover2123) [inline src/parser/WASMParser.cpp:92]: Here, could this default case be met?

File: src/runtime/Value.h [hunk 2]
```diff
@@ -369,7 +369,25 @@ class Value {
...
+    case Value::F32:
+        return sizeof(float);
+    case Value::I64:
+        return sizeof(int64_t);
+    case Value::F64:
+        return sizeof(double);
+    case Value::V128:
+        return 16;
+    default:
+        return sizeof(size_t);
```
Comment (clover2123) [inline src/runtime/Value.h:386]: What about this default case?
Is that any possible case that `type` could be another form?

File: src/parser/WASMParser.cpp (hunk 1 above)
Comment (ksh8281) [reply src/parser/WASMParser.cpp:92]: No. I would add assert not reached

File: src/runtime/Value.h (hunk 2 above)
Comment (ksh8281) [reply src/runtime/Value.h:386]: No. I would add assert not reached

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**